### ISSUE TITLE
ros_ign_bridge with contact sensor fails

### ISSUE THREAD

**Issue by roni-kreinin** [gazebosim/ros_gz#183]
## Environment
* OS Version: Ubuntu 20.04
* Source or binary build?
 Binary, Galactic
    
<details>
  
```
# Failed to create a bridge for topic [/model/robot/bumper_contact] with ROS2 type [ros_ign_interfaces/msg/Contacts] and Ignition Transport type [ignition.msgs.Contacts]
```

</details>

## Description
* Expected behavior: ros_ign_bridge created, bumper_contact topic is published in ros2
* Actual behavior: Bridge fails to be created, nothing is published on ros2

## Steps to reproduce

```
Node(package='ros_ign_bridge', executable='parameter_bridge',
                                 namespace=namespace,
                                 name='bumper_contact_bridge',
                                 output='screen',
                                 parameters=[{
                                     'use_sim_time': use_sim_time
                                 }],
                                 arguments=[
                                     'model/robot/bumper_contact' +
                                     '@ros_ign_interfaces/msg/Contacts' +
                                     '[ignition.msgs.Contacts'
                                 ],
                                 remappings=[
                                     ('model/robot/bumper_contact', '/bumper_contact')
                                 ])
```
I have the bridge working correctly for other sensors, its just contact which is giving me problems.

**Comment by chapulina** [comment]
Bridging the contact message has been added in https://github.com/ignitionrobotics/ros_ign/pull/175, which hasn't been released into Galactic yet, see the changelog: https://github.com/ignitionrobotics/ros_ign/blob/ros2/ros_ign_bridge/CHANGELOG.rst

**Comment by roni-kreinin** [comment]
Is there an ETA for when this will be released? I am having trouble building this package from source, it causes my laptop to freeze for several minutes and gets stuck in progress when I run `colcon build`.

**Comment by mjcarroll** [comment]
#175 has been released as part of galactic `0.233.3 (2021-12-30)`
